src/views/section_title.ts

```diff
@@ -25,6 +25,17 @@ export class SectionTitleView extends BlockView<SectionTitleNode> {
   public contentDOM: HTMLElement
   public elementType = 'h1'
 
+  public createElement = () => {
+    this.contentDOM = document.createElement(this.elementType)
+    this.contentDOM.className = 'block'
+    this.dom.appendChild(this.contentDOM)
+
+    const $pos = this.view.state.doc.resolve(this.getPos())
+    if ($pos.node($pos.depth).type === schema.nodes.bibliography_section) {
```

Comment:
`$pos.node($pos.depth)` is equivalent to `$pos.parent`, right? Could you replace it to make the code more readable?

Reply:
updated